### PR TITLE
Update to latest Rust.

### PR DESCRIPTION
**Pull request by toqueteos**
Waiting for Travis to merge.

### CODE REVIEW

**Comment by toqueteos** [comment]
Changes just merged on gfx_texture were required, retrying Travis.

File: src/texture.rs [hunk 1]
```diff
@@ -191,7 +191,7 @@ impl AtlasBuilder {
     }
 
     /// Returns the complete texture atlas as a texture.
-    pub fn complete<D: Device<C>, C: CommandBuffer>(self, d: &mut D) -> Texture {
-        Texture::from_rgba8(self.image, d)
+    pub fn complete<D: Device>(self, d: &mut D) -> Texture {
+        Texture::from_rgba8::<D>(self.image, d)
```
Comment (eddyb) [inline src/texture.rs]: `::<D>` shouldn't have been required.

Comment (toqueteos) [inline src/texture.rs]: That must be a leftover, my fault.

Comment (toqueteos) [inline src/texture.rs]: Fixed on #33.

**Pull request merged**